src/main/steps/index.ts

```diff
@@ -63,7 +63,13 @@ const getNextIncompleteStep = (
 export const getNextIncompleteStepUrl = (req: AppRequest): string => {
   const { queryString } = getPathAndQueryString(req);
   const sequence = getUserSequence(req);
-  const url = getNextIncompleteStep(req.session.userCase, sequence[0], sequence, true);
+  const sequenceIndex =
+    !req.session.isApplicant2 &&
+    (req.session.userCase.state === State.ConditionalOrderDrafted ||
+      req.session.userCase.state === State.ConditionalOrderPending)
+      ? sequence.findIndex(s => s.url.includes(READ_THE_RESPONSE))
+      : 0;
```

Comment:
is this because theres a break in the sequence?

i think if thats the case you can move the urls closer to match the actual sequence as i know sometimes we just add it to the end which might break this and this will be unnecessary.